model/generate.go

```diff
@@ -270,10 +270,16 @@ func (g *GeneratedProject) saveNewBuildsAndTasks(ctx context.Context, v *Version
 		return errors.Errorf("project '%s' not found", p.Identifier)
 	}
 
+	allTasksToBeCreatedIncludingDeps, err := NewPatchTaskIdTable(p, v, *newTVPairs, projectRef.Identifier)
```

Comment:
Tangential but it seems `NewPatchTaskIdTable` can run even for mainline commits that use generate.tasks, we should probably rename the function to something that doesn't imply it's just for patches, like `NewTaskIdTable`